src/aat/resources/features/F-108/F-108.feature

```diff
@@ -0,0 +1,268 @@
+    Given a case that has just been created as in [Private_Case_Creation_Autotest1_Data]
+    And logstash has finished indexing case data
+    And a user with [a role with read access to the case state]
+    When the request [is configured to search for the previously created case via exact match]
+    And a request is prepared with appropriate values
+    And it is submitted to call the [external search query] operation of [CCD Data Store Elastic Search API]
+    Then a positive response is received
+    And the response [contains the previously created case data]
+    And the response has all other details as expected
+
+
+    @S-604
+  Scenario: should NOT return the case for a role with no read access to a case state
+    Given a case that has just been created as in [Private_Case_Creation_Autotest1_Data]
+    And logstash has finished indexing case data
+    And a user with [a role with no read access to the case state]
+    When the request [is configured to search for the previously created case via exact match]
+    And a request is prepared with appropriate values
+    And it is submitted to call the [external search query] operation of [CCD Data Store Elastic Search API]
+    Then a positive response is received
+    And the response [contains no cases]
+    And the response has all other details as expected
+
+
+    @S-605
+  Scenario: should return the case field where user role matches ACL and security classification
+    Given a case that has just been created as in [Private_Case_Creation_Autotest1_Data]
+    And logstash has finished indexing case data
+    And a user with [a role with security classification of RESTRICTED]
+    When the request [is configured to search for the previously created case via exact match]
+    And a request is prepared with appropriate values
+    And it is submitted to call the [external search query] operation of [CCD Data Store Elastic Search API]
+    Then a positive response is received
+    And the response [contains the RESTRICTED email field value]
+    And the response has all other details as expected
+
+  ### CrossCaseTypeSearch
+    @S-610
+  Scenario: should return cases only for case types the user has access to - the user role can read case type and has same security classification "
+  + "as case type
+    Given a case that has just been created as in [S-610_Create_Case_Private_Autotest1]
+    And a case that has just been created as in [S-610_Create_Case_Private_Autotest2]
+    And logstash has finished indexing case data
+    And a user with [private access to AUTOTEST1 jurisdiction only]
+    When the request [is configured to search for both the previously created cases]
+    And a request is prepared with appropriate values
+    And it is submitted to call the [external search query] operation of [CCD Data Store Elastic Search API]
+    Then a positive response is received
+    And the response [contains only 1 case]
```

Comment:
Expressly tell which case among the 2 above is expected to be in the response, to make the wording firmer to describe the requirement just in AC script, without relying on reading of the test data.